cmd/tempo/app/app.go

```diff
@@ -217,7 +218,7 @@ func (t *App) Run() error {
 		// let's find out which module failed
 		for m, s := range serviceMap {
 			if s == service {
-				err = service.FailureCase()
+				err := service.FailureCase()
```

Comment:
Drive-by change; I ran my test with `-race` and it showed me a data race here with the other `err` outside of this goroutine.